.env.test

```diff
@@ -7,4 +7,5 @@ DATABASE_PORT=5432
 REDIS_URL=redis://solutions_redis:6379/12
 APP_HOST=lvh.me:5250
 RECAPTCHA_SITE_KEY=6Lc6BAAAAAAAAChqRbQZcn_yyyyyyyyyyyyyyyyy
-RECAPTCHA_SECRET_KEY=6Lc6BAAAAAAAAKN3DRm6VA_xxxxxxxxxxxxxxxxx
\ No newline at end of file
+RECAPTCHA_SECRET_KEY=6Lc6BAAAAAAAAKN3DRm6VA_xxxxxxxxxxxxxxxxx
+RAILS_SECRET='a_random_string_key_of_very_long_length'
```

Comment:
@ajaystha42 set this in https://github.com/restarone/violet_rails/blob/13a9f79d8abec3c6c72061043c6530b5232175e6/config/secrets.yml 

and then rename `RAILS_SECRET` to `SECRET_KEY_BASE`